src/ucs/stats/stats.h

```diff
@@ -19,10 +19,61 @@ BEGIN_C_DECLS
 
 /** @file stats.h */
 
+/* The maximum size of a class name */
+#define MAX_CLS_NAME_SIZE 50
+
+/* The maximum number of UCX aggregate-sum counters */
+#define MAX_NUM_OF_STATS_AGGREGATE_SUM_COUNTERS 32
+
+/* The maximum number of statistics classes */
+#define MAX_NUM_OF_STATS_CLASSES 32
+
+typedef struct tag_ucs_stats_counter {
+    /* Counter class name */
+    const char *class_name;
+
+    /* Counter name */
+    const char *name;
+
+    /* Counter value */
+    uint64_t value;
+
+} ucs_stats_counter_s;
+
+typedef struct tag_ucs_stats_cls_id_to_name {
+    /* Class name */
+    char class_name[MAX_CLS_NAME_SIZE];
+
+    /* Class base offset in the counters aggregate-sum list */
+    unsigned aggregate_sum_class_offset;
+
+} ucs_stats_class_id_to_name_s;
```

Comment:
struct naming: should start with ucs_stats, typedefs end with _t
code style: indent on column
2 space lines between structs in H file

Reply:
Fixed.
Question, what do you mean by: "code style: indent on column"?

Reply:
Nevermind, fixed.